src/core/path_coord.cpp

```diff
@@ -506,9 +511,10 @@ SplitPathCoord SplitPathCoord::at(
 		{
 			--split.path_coord_index;
 		}
+		
+		split.index = path_coords[split.path_coord_index].index;
 	}
 	
-	split.index = path_coords[split.path_coord_index].index;
```

Comment:
Drive-by change. When not entering the `if` branch, the right `split.index` is implied by initialization of `split`.